### ISSUE TITLE
Adding YR to Left Nav (IA Recs from Content Team)

### ISSUE THREAD

**Issue by AkucherGCIO** [department-of-veterans-affairs/va.gov-team#6107]
## User story
As a Veteran interested in using the Yellow Ribbon tool, I need to know where to find it in the left nav.

## Issue description

Add Yellow Ribbon tool to Education let nav under More Resources spoke with wording "Yellow Ribbon participating schools"

## Tasks
- [x] Confirm left nav placement with Mikki
- [x] Add YR tool to left nav

## Acceptance criteria
- [x] Yellow Ribbon program tool visible in left nav on live site

**Comment by AkucherGCIO** [comment]
1. Need to add more detail after discussing with Randi.
2. Need to add blocked by issue.

**Comment by RLHecht** [comment]
@stephorkandatx Not sure if this work is included in any of the other YR tickets that are currently floating around. Should it be closed or kept open?

**Comment by RLHecht** [comment]
@stephorkandatx Now that YR is live, we should move this ticket into the next sprint. I know we need to add a link to the tool but not sure where exactly (under where we list the main YR page as a child of the Post-9/11 GI Bill?). Let me know if I should follow up with Mikki.

**Comment by RLHecht** [comment]
@mnorthuis I'd like to had the YR tool to the left nav but need more information on where it should be placed and the wording. Is there another ticket that references the work I can see?

**Comment by mnorthuis** [comment]
@RLHecht Entry points documented here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/websites-yellow-ribbon.md

**Comment by RLHecht** [comment]
@AnneHurley and Beth (I can't tag here so will slack ticket!), when adding the YR tool link to left nav I want to confirm that the text should be "Yellow Ribbon participating schools"? This was suggested in Mikki's doc above.

**Comment by AnneHurley** [comment]
@RLHecht  and Beth (will also tag) -- We're typically using "Find a Yellow Ribbon school" for the page title and how we refer to the page. I'd recommend that also be the left nav wording. @mnorthuis  Making sure you're ok with that?

**Comment by mnorthuis** [comment]
We haven't generally used action words with content inside resources, we even changed "Compare GI Bill schools" to "GI Bill school comparison".  I realize that's what I had documented in my notes for H1 as well, but we should double check on this for consistency.  The URL is intentionally not action based for this reason.

**Comment by bethpottsVADEPO** [comment]
@mnorthuis is right about the approach we've been using with left nav. It doesn't always match the H1--and under "Get benefits" in the left nav our labels don't start with a verb. I think we could shorten it further to: Yellow Ribbon schools.

**Comment by RLHecht** [comment]
This has been added to the left nav as "Yellow Ribbon schools." Should be live after today's deploy. Moving to validate. ![Screen Shot 2020-06-04 at 9.38.46 AM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/0738f364-9b87-4be0-a542-23f7d4ef5d28)

**Comment by RLHecht** [comment]
This is validated on live site. Closing ticket! FYSA @stephorkandatx @bethpottsVADEPO @AnneHurley
